core/templates/dev/head/components/AnswerGroupEditorDirective.js

```diff
@@ -55,6 +55,9 @@ oppia.directive('answerGroupEditor', [
           $scope.misconceptions = StateEditorService.getMisconceptions();
 
           var _getTaggedMisconceptionName = function(misconceptionId) {
+            if (!$scope.misconceptions) {
```

Comment:
Is this a list? If so check its length instead. In JS, [] is considered "true".

Reply:
Done.